src/spec-configuration/containerFeaturesOCI.ts

```diff
@@ -52,7 +52,7 @@ export function getOCIFeatureSet(output: Log, identifier: string, options: boole
             type: 'oci',
             manifest: manifest,
             featureRef: featureRef,
-
+            referenceId: `${featureRef.resource}:${featureRef.version}`
```

Comment:
I believe `identifier` here is exactly the value you're looking for.

This is the only current use of that function 
```
let newFeaturesSet: FeatureSet = getOCIFeatureSet(output, userFeature.id, userFeature.options, manifest);

```

Reply:
Ah! right 👍